packages/react-devtools-extensions/package.json

```diff
@@ -16,6 +16,7 @@
     "test:chrome": "node ./chrome/test",
     "test:firefox": "node ./firefox/test",
     "test:edge": "node ./edge/test",
+    "improve-images": "node ./scripts/images/improveImages.mjs",
```

Comment:
nit: Other scripts for this package (e.g. `build.js` or `deploy.js`) just live in the root directory (beside `package.json`). Probably good to just follow that convention rather than introduce a new one?

Reply:
In other words: we import `scripts/images/improveImages.mjs` in `build.js` and run it there?